generator/src/generator/namespace.rs

```diff
@@ -2353,7 +2353,7 @@ impl<'ns, 'c> NamespaceGenerator<'ns, 'c> {
 
         let (raw_type, smaller_types, larger_types): (&str, &[&str], &[&str]) =
             match global_enum_size {
-                1 => ("bool", &[], &[]),
+                1 => ("bool", &[], &["u8", "u16", "u32", "u64"]),
```

Comment:
Is the `u64` useful? If so, should it be added for the other cases too?

Reply:
This catch. I was just typing away without looking at the other cases. Fixed now.